fedmsg_meta_fedora_infrastructure/coprs.py

```diff
@@ -17,9 +17,32 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 
+import copy
+
 from fedmsg_meta_fedora_infrastructure import BaseProcessor
 from fasshim import gravatar_url
 
+_statuses = {
+    0: 'failed',
+    1: 'success',
+    3: 'running',
+    5: 'skipped',
+}
+
+
+_long_template = """Package:  {pkg}
+COPR:     {owner}/{copr}
+Built by: {user}
+Status:   {status}
+ID:       {build}
+
+Logs:
+  Build:     http://copr-be.cloud.fedoraproject.org/results/{owner}/{copr}/{chroot}/build-{build}.log
+  Mockchain: http://copr-be.cloud.fedoraproject.org/results/{owner}/{copr}/{chroot}/mockchain.log
```

Comment:
Not sure if mockchain is useful (but there's no harm having it of course), since it's mock chain of all builds ever done in the copr.

Link to root.log (from mock) would be useful though:

http://copr-be.cloud.fedoraproject.org/results/{owner}/{copr}/{chroot}/{pkg}/root.log